### PR TITLE
Add code upload cluster instance config in model

### PR DESCRIPTION
**Pull request by Ram81**
### Description

This PR -

- [x] Adds changes to configure code upload cluster instance setting from challenge model fields.

### CODE REVIEW

File: apps/challenges/aws_utils.py [hunk 1]
```diff
@@ -1094,11 +1094,15 @@ def create_eks_nodegroup(challenge, cluster_name):
         response = client.create_nodegroup(
             clusterName=cluster_name,
             nodegroupName=nodegroup_name,
-            scalingConfig={"minSize": 1, "maxSize": 10, "desiredSize": 1},
-            diskSize=100,
+            scalingConfig={
+                "minSize": 1,
+                "maxSize": challenge_obj.max_worker_instance,
+                "desiredSize": 1,
```
Comment (RishabhJain2018) [inline apps/challenges/aws_utils.py:1100]: Can we please add this to the database as well?

File: apps/challenges/aws_utils.py [hunk 2]
```diff
@@ -1094,11 +1094,15 @@ def create_eks_nodegroup(challenge, cluster_name):
         response = client.create_nodegroup(
             clusterName=cluster_name,
             nodegroupName=nodegroup_name,
-            scalingConfig={"minSize": 1, "maxSize": 10, "desiredSize": 1},
-            diskSize=100,
+            scalingConfig={
+                "minSize": 1,
```
Comment (RishabhJain2018) [inline apps/challenges/aws_utils.py:1098]: Can we please add this to the database as well?

File: apps/challenges/models.py [hunk 3]
```diff
@@ -152,6 +152,17 @@ def __init__(self, *args, **kwargs):
...
         null=True, blank=True, max_length=200, default=""
     )
+    # Evaluation instance config for code upload challenge
+    worker_instance_type = models.CharField(
+        max_length=256, null=True, blank=True, default="g4dn.xlarge"
+    )
+    worker_ami_type = models.CharField(
+        max_length=256, null=True, blank=True, default="AL2_x86_64_GPU"
+    )
+    worker_disk_size = models.IntegerField(null=True, blank=True, default=500)
```
Comment (RishabhJain2018) [inline apps/challenges/models.py:162]: Please change the default to 100.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

File: apps/challenges/serializers.py [hunk 4]
```diff
@@ -248,6 +248,10 @@ class Meta:
             "vpc_cidr",
             "subnet_1_cidr",
             "subnet_2_cidr",
+            "worker_instance_type",
+            "worker_ami_type",
+            "worker_disk_size",
+            "max_worker_instance",
```
Comment (RishabhJain2018) [inline apps/challenges/serializers.py:254]: Can we please add `max_worker_instance` and `desired_worker_instance` here as well.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

**Comment by Ram81** [comment]
@RishabhJain2018 done

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**